docs/Reflection/Reflection.tex

```diff
@@ -14,39 +14,126 @@
+
+\section{Changes in Response to Feedback}
+
+\plt{Summarize the changes made over the course of the project in response to
+feedback from TAs, the instructor, teammates, other teams, the project
+supervisor (if present), and from user testers.}
+
+\plt{For those teams with an external supervisor, please highlight how the feedback 
+from the supervisor shaped your project.  In particular, you should highlight the 
+supervisor's response to your Rev 0 demonstration to them.}
+
+\subsection{SRS and Hazard Analysis}
+
+\subsection{Design and Design Documentation}
+
+\subsection{VnV Plan and Report}
+
+\section{Design Iteration (LO11)}
+
+\plt{Explain how you arrived at your final design and implementation.  How did
+the design evolve from the first version to the final version?} 
+
+\noindent The final design and implementation contain two pieces: the front-end, which is the client, and the back-end, which is the server. As the client is user-facing and directly interacts with it, the design is swayed by feedback and how the application "feels" to use. Whereas the server is not directly interacted with by the users, they only see the results of it. Given this, the client was changed more than the server.
+
+\subsection{Client Design and Implementation}
+Going deeper into the client's design, we, as developers, are also users of the application, where we had a decent idea of how the client should look and the overall flow of it, which is what we started with. Using Figma, 7eam could display their thoughts as pictures, where each component should be, and what pages are needed. Going from Figma to the actual initial implementation had a few colour changes and made the application feel smooth. However, after publishing the application and collecting feedback, more changes had to be made:
+\begin{itemize}
+	\item Additional preferences were added
+	\item History functionality was added
+	\item Layout was modified on some pages
+	\item Adding friends is clunky
+\end{itemize}
+Making the difference between us as developers and users abundantly clear. We were extremely comfortable as we had been looking at the pages endlessly, but a user coming in for the first time had to learn the layout. 7eam wanted to make that learning process seamless and to use the application faster. Implementing and designing the client is a constant reiterative process. Users can provide more detailed feedback on what they expect as they use the application more. The client's current iteration cannot be considered the final and is expected to evolve continually. Still, the abovementioned changes are how the client changed from the first iteration to the current one.
```

Comment:
7eam wanted to make that learning process seamless and to use the application faster. Replace with "7eam wanted to make sure that the users were able to learn how to use the application in a fast and simple way."

Reply:
"current iteration cannot be considered **~~the~~** final"